src/frontend/screens/Settings/components/OfflineMode.tsx

```diff
@@ -26,17 +28,28 @@ const OfflineMode = () => {
 
   const [offlineMode, setOfflineMode] = useSetting('offlineMode', false)
 
-  if (isDefault || !canRunOffline) {
+  if (isDefault) {
```

Comment:
This feature is for Epic games only, right? because GOG will work fine offline since they don't have DRM. Same for Amazon games. So we could hide it for GOG or disable it and show a information about it.

Reply:
I changed this to only show the option for Epic games.